src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -55,21 +57,9 @@ If you're brand new to New Relic and want to see how easy using NRQL can be, you
   </Collapser>
 </CollapserGroup>
 
-## Syntax [#syntax]
-
-This resource contains explanations for the various functions and clauses available for use in a NRQL query. Other popular resources for understanding NRQL include:
-
-* [Intro to NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql): provides a brief overview of NRQL as a whole, and links to other introductory docs
-* [How to use NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql-how-nrql-works): explains how NRQL works and how to use it to query data
-* [Introductory NRQL tutorial](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql-tutorial): a basic tutorial for NRQL designed to get you familiar with basic queries
-* [NRQL instructional course](https://learn.newrelic.com/writing-nrql-queries) from New Relic University
-* [Learn how to query the `Metric` data type](/docs/telemetry-data-platform/get-data/apis/query-metric-data-type)
-* [Use subqueries](/docs/query-your-data/nrql-new-relic-query-language/get-started/subqueries-in-nrql)
-* [Use funnels to evaluate a series of related data](/docs/insights/new-relic-insights/features/funnels)
-
 ## Query components [#clauses]
 
-Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All other clauses are optional. The clause definitions below also contain example NRQL queries.
+As noted in our [basic NRQL syntax doc](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/#syntax), every NRQL query will contain a `SELECT` clause and a `FROM` clause, and will start with one of those. All other clauses are optional. The clause definitions below also contain example NRQL queries.
```

Comment:
Since NRQL Variables were introduced, you can also start a query with WITH (e.g. https://onenr.io/07jbm0AgWRy).  Looks like we missed this spot when updating the docs.

Reply:
ok thanks